static/js/settings_linkifiers.js

```diff
@@ -142,7 +142,9 @@ export function build_page() {
                         ui_report.error(i18n.t("Failed"), xhr, pattern_status);
                     }
                     if (errors.url_format_string !== undefined) {
-                        xhr.responseText = JSON.stringify({msg: errors.url_format_string});
+                        xhr.responseText = JSON.stringify({
+                            msg: errors.url_format_string[0] + " " + errors.url_format_string[1],
+                        });
```

Comment:
Hmm, can you add a comment with an example of what the `url_format_string` response looks like?

Reply:
I have added the before and after Screenshots above, i should comment in that too ? @timabbott

Reply:
Yes, I see what it looks like visually -- I mean what format the server sends here.

Reply:
![(135) Private messages - Zulip Community - Zulip - Google Chrome 04-04-2021 09_24_00](https://user-images.githubusercontent.com/77020164/113498167-23653d80-9528-11eb-8c28-0480e0b029f1.png)